### ISSUE TITLE
[Notifier] Text contrast does not meet a11y criteria for normal text

### ISSUE THREAD

**Issue by dpayk** [open-pioneer/trails-openlayers-base-packages#153]
For normal text, the contrast ratio is under 1:4.5 for all four colors that are used as background in the notifier.
